core/src/test/scala/com/microsoft/azure/synapse/ml/explainers/split1/ICEExplainerSuite.scala

```diff
@@ -53,6 +53,30 @@ class ICEExplainerSuite extends TestBase with TransformerFuzzing[ICETransformer]
     .setKind("average")
   lazy val outputAvg: DataFrame = iceAvg.transform(data).cache()
 
+  lazy val iceFeat: ICETransformer = new ICETransformer()
+    .setModel(model)
+    .setTargetCol("probability")
+    .setCategoricalFeatures(Array(ICECategoricalFeature("col1", Some(100)), ICECategoricalFeature("col2"),
+     ICECategoricalFeature("col3")))
+    .setNumericFeatures(Array(ICENumericFeature("col4", Some(5))))
+    .setTargetClasses(Array(1))
+    .setKind("feature")
+  lazy val outputFeat: DataFrame = iceFeat.transform(data).cache()
+  // Output schema: number_of_features rows * 2 cols (name of the feature + corresponding dependence)
+  // Output is sorted by dependence in descending order
+  // For this example you should expect:
+  //  outputFeat.show(false)
+ //  +------------------------+---------------+
+//  |dependence              |featureNames   |
```

Comment:
if featureNames is an output column perhaps we should provide a getter/setter and treat this and other columns like it as a parameter. featureNames can be the default value of this param if not set